rvps/src/extractors/extractor_modules/swid/mod.rs

```diff
@@ -85,23 +75,15 @@ impl Extractor for SwidExtractor {
                     break;
                 }
 
-                let rv_pair = HashValuePair::new(DEFAULT_ALG.into(), hash.unwrap().to_string());
+                let hash_value = serde_json::Value::String(hash.unwrap().to_string());
```

Comment:
we can use the following structure to avoid `unwrap`
```rust
let Some(hash) = resource.attribute((HASH_NS, format!("Hash{hash_index}").as_str())) else {
    break;
}
// ... use `hash` variable
```